packages/roosterjs-editor-plugins/lib/PasteManager/PasteManager.ts

```diff
@@ -94,25 +102,70 @@ export default class PasteManager implements EditorPlugin {
     };
 }
 
-function getOnPasteHandler(): OnPasteHandler {
-    if (browserData.isChrome || browserData.isFirefox || browserData.isEdge) {
-        return onPaste;
-    } else if (browserData.isIE) {
-        return onPasteIE;
-    } else if (browserData.isSafari) {
-        return onPasteSafari;
+function getClipboardData(dataTransfer: DataTransfer): ClipboardData {
+    let clipboardData: ClipboardData = {
+        imageData: {},
+        textData: dataTransfer.getData('text'),
+        htmlData: null,
+    };
+
+    let image = getImage(dataTransfer);
+    if (image) {
+        clipboardData.imageData.file = image;
+    }
+
+    return clipboardData;
+}
+
+function getImage(dataTransfer: DataTransfer): File {
+    // Chrome, Firefox, Edge support dataTransfer.items
+    let fileCount = dataTransfer.items ? dataTransfer.items.length : 0;
+    for (let i = 0; i < fileCount; i++) {
+        let item = dataTransfer.items[i];
+        if (item.type && item.type.indexOf('image/') == 0) {
+            return item.getAsFile();
+        }
+    }
+    // IE, Safari suuport dataTransfer.files
+    fileCount = dataTransfer.files ? dataTransfer.files.length : 0;
+    for (let i = 0; i < fileCount; i++) {
+        let file = dataTransfer.files.item(i);
+        if (file.type && file.type.indexOf('image/') == 0) {
+            return file;
+        }
+    }
+    return null;
+}
+
+function getHtml(editor: Editor, callback: (container: HTMLElement) => void) {
```

Comment:
The function name is a little confused. Usually a function starts with "get" will return something, but this method has no return value. Suggest to change the function name.